servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -106,10 +132,39 @@ void RenderForwardMobile::RenderBufferDataForwardMobile::configure(RID p_color_b
 
 		{
 			Vector<RID> fb;
-			fb.push_back(color_msaa);
-			fb.push_back(depth_msaa);
+			fb.push_back(color_msaa); // 0
+			fb.push_back(depth_msaa); // 1
+			fb.push_back(color); // 2
+			// fb.push_back(p_final_color_buffer); // 3
+
+			// Now define our subpasses
+			Vector<RD::FramebufferPass> passes;
+			RD::FramebufferPass pass;
+
+			// re-using the same attachments
+			pass.color_attachments.push_back(0);
+			pass.depth_attachment = 1;
+
+			// - opaque pass
+			passes.push_back(pass);
+
+			// - sky pass
+			passes.push_back(pass);
 
-			color_fb = RD::get_singleton()->framebuffer_create(fb, RenderingDevice::INVALID_ID, view_count);
+			// We may need to finalise our pass here and start a new pass if we have shaders that use SCREEN_TEXTURE or DEPTH_TEXTURE
+			// I don't think we support this at this time...
+
+			// - alpha pass
+			pass.resolve_attachments.push_back(2); // this does not work..
+			passes.push_back(pass);
```

Comment:
@reduz is this what you're expecting to see to do the MSAA resolve? This does not seem to work for me.

Reply:
This now works, turned out to require a special bit to be set on the texture.